src/pages/demos/hero.hbs

```diff
@@ -0,0 +1,173 @@
+---
+title: Page Hero examples
+layout: blank
+styles:
+  - hero
+---
+
+
+{{#embed "patterns.organisms.hero.hero"}}
+  {{#content "content"}}
+    <h1 class="mzp-c-hero-title">A Basic Page Hero</h1>
+
+    <div class="mzp-c-hero-desc">
+      <p>A description with approximately 140 characters. That means we usually only have room for one or two sentences. This is what that looks like.</p>
+    </div>
+
+    <p class="mzp-c-hero-cta">
+      <a class="mzp-c-button mzp-c-button-primary" href="#">Call to action</a>
+    </p>
+  {{/content}}
+{{/embed}}
+
+<hr>
+
+{{#embed "patterns.organisms.hero.hero" image="true"}}
+  {{#content "content"}}
+      <h1 class="mzp-c-hero-title">A Page Hero with Image</h1>
+
+      <div class="mzp-c-hero-desc">
+        <p>A description with approximately 140 characters. That means we usually only have room for one or two sentences. This is what that looks like.</p>
+      </div>
+
+      <p class="mzp-c-hero-cta">
+        <a class="mzp-c-button mzp-c-button-primary" href="#">Call to action</a>
+      </p>
+    {{/content}}
+
+  {{#content "hero-image"}}
+  <div class="mzp-c-hero-image">
+    <img src="https://placeimg.com/1200/675/nature" alt="">
```

Comment:
I'm not entirely confortable with using a third party service to deliver placeholder images (even though they are likely all safe-for-work 😄). It would be nice to incorporate some more Firefox/Mozilla specific imagery into the demo, if we have something?

Reply:
I was using placeimg as an easy way to test lots of different sizes and aspect ratios and intended to replace them with suitable local images, but was rushing to get this PR ready for review before I go missing for 36 hours. But I can swap in different images this morning.